app/assets/src/components/ui/containers/ColumnHeaderTooltip.jsx

```diff
@@ -14,6 +14,10 @@ class ColumnHeaderTooltip extends React.Component {
               <span className={cs.title}>{this.props.title}:</span>
             )}
             {this.props.content}
+            {this.props.list &&
```

Comment:
nit: Can we move loading the props taking advantage and refactor the other properties/